docs/admin/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+id: overview
+title: "Administration"
+sidebar_label: "Administration"
+---
+
+The admin section of Scoreboard allows admin users to manage badges, teams, and users.
+
+## [Badges](./badges)
+
+- [Create new badges](./create-badge)
```

Comment:
All of these links should be `../<page-name>` because as is the links redirect to `docs/admin/overview/<page-name>` as opposed to `docs/admin/<page-name>`